plugins/source/gcp/client/spec/spec.go

```diff
@@ -56,7 +56,6 @@ func (spec *Spec) SetDefaults() {
 }
 
 // JSONSchemaExtend is required to add `not` section for `project_filter` & `folder_ids` being mutually exclusive.
-// We use value receiver because of https://github.com/invopop/jsonschema/issues/102
```

Comment:
GCP change? should be a `chore` in a separate PR

Reply:
it's only a comment replacement, I'll remove this one now (& move it to a chore)

Reply:
done in 3ef9094

Reply:
yeah but being in the `gcp` this PR would get included in the GCP changelog, version would increase (because of `feat` and so on)